javascript/sdk/lib/types.ts

```diff
@@ -275,10 +273,9 @@ export interface ActivityQuery {
   control?: ControlCode;
 }
 
-export interface DayResult {
-  count: number;
-  datetime: string;
-  failed: number;
+export interface DayActivity {
```

Comment:
it looks like there's still a `datetime` field 
`days.append(dict(datetime=h[0], unprotected=h[1], volume=h[2]))`

Reply:
re-added :)